### PR TITLE
Use admin.kubeconfig when obtaining kubernetes service ip

### PR DESCRIPTION
**Pull request by abutcher**
http://post-office.corp.redhat.com/archives/openshift-sme/2015-November/msg01139.html

@twiest @wshearn

### CODE REVIEW

**Comment by wshearn** [comment]
:+1:

File: playbooks/common/openshift-master/config.yml [hunk 1]
```diff
@@ -355,7 +355,7 @@
   hosts: oo_first_master
   tasks:
   - name: Get master service ip
-    command: "{{ openshift.common.client_binary }} get -o template svc kubernetes --template=\\{\\{.spec.clusterIP\\}\\}"
+    command: "{{ openshift.common.client_binary }} -n default --config={{ openshift.common.config_base }}/master/admin.kubeconfig get -o template svc kubernetes --template=\\{\\{.spec.clusterIP\\}\\}"
```
Comment (detiber) [inline playbooks/common/openshift-master/config.yml]: I have no objections to using that admin kubeconfig here...  I just don't want to see us use it for anything that could mutate the config.

**Comment by abutcher** [comment]
Fixes #994

**Comment by abutcher** [comment]
@brenton

**Pull request merged**